templates/challenge.html

```diff
@@ -194,23 +200,31 @@
   </div>
 
   <script type="text/javascript">
+    let initTheme = localStorage.getItem('theme') === 'dark' ? "material-darker" : "default"
     let codeMirrorShared = {
```

Comment:
`codeMirrorOptionsShared`

Reply:
Maybe we can rename this variable to `sharedCodeMirrorOptions` or `sharedCodeMirrorOpts`？🤔

Reply:
`sharedCodeMirrorOptions` feels better